securedrop/journalist_templates/col.html

```diff
@@ -24,8 +24,8 @@
       <p>
         <div id="select-container"></div>
         <button type="submit" name="action" value="download" class="small"><i class="fa fa-download"></i> {{ gettext('Download Selected') }}</button>
-        <a href="#delete-selected-confirmation-modal" id="delete-selected-link" class="btn small danger">
```

Comment:
could you please add to the commit message a short description of the problem for posterity ? It is unclear to me what removing the btn class solves.

Reply:
So I didn't exactly remove the btn class. I created an entire button inside the <a> tag instead of making the <a> tag look like a button with the "btn" class. I did this because without it, in smaller screen, there was a partial wrapping rather than the entire button going to the next line as seen [here](https://github.com/freedomofpress/securedrop/pull/2974#issuecomment-363206840)

I will add a commit message for the same.

Reply:
Oh, I overlooked that part of the diff, thanks for explaining. It makes total sense now. Good to merge once the commit message is amended so it is cristal clear to future readers.

Reply:
@dachary updated the commit messages. Please check if it looks fine.